### PR TITLE
docs(fromEvent): update the example to match the actual API

### LINKED ISSUE #7245: FromEvent Docs: No overload matches this call.
### Describe the bug

As per the https://rxjs.dev/api/index/function/fromEvent in order to use this operator with capture option we can pass the three arguments to it. 
image
But LSP is showing the following error.
image

### Expected behavior

As per the documentation.
**DOM EventTarget**

This is an object with addEventListener and removeEventListener methods.

In the browser, addEventListener accepts - apart from event type string and event handler function arguments - **optional third parameter, which is either an object or boolean, both used for additional configuration how and when passed function will be called.** When [fromEvent](https://rxjs.dev/api/index/function/fromEvent) is used with event target of that type, you can provide this values as third parameter as well.

### Reproduction code

```typescript
Open the following example in Docs using StackBlitz "Use addEventListener with capture option" (https://rxjs.dev/api/index/function/fromEvent)
```

### Reproduction URL

_No response_

### Version

~7.5.0

### Environment

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by demensky**
Updates the example according to the API.

**Related issue (if exists):**
Closes #7245

### CODE REVIEW

File: src/internal/observable/fromEvent.ts [hunk 1]
```diff
@@ -204,8 +204,8 @@ export function fromEvent<T, R>(
  * ```ts
  * import { fromEvent } from 'rxjs';
  *
- * const clicksInDocument = fromEvent(document, 'click', true); // note optional configuration parameter
- *                                                              // which will be passed to addEventListener
+ * // note optional configuration parameter which will be passed to addEventListener
+ * const clicksInDocument = fromEvent(document, 'click', { capture: true });
  * const clicksInDiv = fromEvent(someDivInDocument, 'click');
```
Comment (jakovljevic-mladen) [inline src/internal/observable/fromEvent.ts:209]: This line fails as well. Can you please create a `div` so it doesn't fail? There are many examples in code base, e.g.

```ts
const div = document.createElement('div');
div.style.cssText = 'width: 200px; height: 200px; background: #09c;';
document.body.appendChild(div);
```

```suggestion
 * const clicksInDiv = fromEvent(div, 'click');
```

**Review by jakovljevic-mladen — COMMENTED** [review]
LGTM, only a minor suggestion so that this example becomes fully functional.

File: src/internal/observable/fromEvent.ts (hunk 1 above)
Comment (demensky) [reply src/internal/observable/fromEvent.ts:209]: @jakovljevic-mladen something like this?

```diff
import { fromEvent } from 'rxjs';
+
+ const div = document.createElement('div');
+ div.style.cssText = 'width: 200px; height: 200px; background: #09c;';
+ document.body.appendChild(div);

// note optional configuration parameter which will be passed to addEventListener
const clicksInDocument = fromEvent(document, 'click', { capture: true });
- const clicksInDiv = fromEvent(someDivInDocument, 'click');
+ const clicksInDiv = fromEvent(div, 'click');

clicksInDocument.subscribe(() => console.log('document'));
clicksInDiv.subscribe(() => console.log('div'));
```

Comment (jakovljevic-mladen) [reply src/internal/observable/fromEvent.ts:209]: Yes.

Comment (demensky) [reply src/internal/observable/fromEvent.ts]: Done.

**Comment by jakovljevic-mladen** [comment]
@demensky, can you please update PR by changing target to `master` branch? After merge, I usually manually cherry-pick to `7.x`, I don't do that the other way around.

**Review by jakovljevic-mladen — APPROVED** [review]
LGTM. Thanks for changing target to the `master`.

**Pull request merged**